CHANGELOG.md

```diff
@@ -4,9 +4,11 @@ x.x.x Release notes (yyyy-MM-dd)
 * None.
 
 ### Fixed
-* <How to hit and notice issue? what was the impact?> ([#????](https://github.com/realm/realm-js/issues/????), since v?.?.?)
 * Realm JS can now be installed in environments using npm binary mirroring ([#4672](https://github.com/realm/realm-js/pull/4672), since v10.0.0).
-* None.
+* Asymmetric sync now works with embedded objects. ([realm/realm-core#5565](https://github.com/realm/realm-core/issues/5565), since 10.19.0)
+* Fixed an issue on Windows that would cause high CPU usage by the sync client when there are no active sync sessions. ([realm/realm-core#5591](https://github.com/realm/realm-core/issues/5591), since v1.1.1)
```

Comment:
>since v1.1.1
Is this the correct version number?

Reply:
The core entry is "since the introduction of Sync support for Windows", and Windows support to Realm JS was added in v1.1.1 and sync is mentioned in v1.0.0. So my guess is that it has always been an issue.